src/vos/ilog.h

```diff
@@ -47,17 +47,18 @@ struct umem_instance;
 
 enum ilog_status {
 	/** Log entry is visible to caller */
```

Comment:
Do the comments need to change with these?

Reply:
I don't think this is the last iteration on this...for now, I leave them